README.md

```diff
@@ -38,73 +44,34 @@ If you make changes to the [Domain Verification Checker](https://xrpl.org/valida
-
-When you're done translating, [compile the PO files](#compile-strings).
-
-### Update Strings
-
-If there are new or updated `{% trans %}` tags in the templates, first use this command to extract them:
-
-```sh
-$ pybabel extract -F ./locale/babel.cfg -o ./locale/messages.pot ./
-```
-
-Then, update _every_ language's `.po` files with the list of strings, as follows:
-
-```sh
-$ pybabel update -l ja -d ./locale/ -i ./locale/messages.pot
-```
+        npm start
 
-The above example is for Japanese (`-l ja`). **Repeat for each language code.**
 
-Now edit the PO files (for example, `locale/ja/LC_MESSAGES/messages.po`) to add translations for each newly-added string. Again, **repeat for each language.**
+### Internationalization 
 
-If you only want to change an existing translation for a given string that hasn't changed in the original, you can skip straight to editing the PO files without running any `update` or `extract` commands.
+This repo has English (en) and Japanese (ja) locales set up already. 
 
-After you've edited all the PO files, be sure to [compile them](#compile-strings).
+This is done by setting up the internationalization (@i18n) folders, adding the `i18n` configuration to your `redocly.yaml` file, and adding the translated content in the respective language directory under the @i18n directory.
 
-### Compile Strings
+To add support for a new language:
 
-Whether you added a language, added new strings, or tweaked an existing translation, you must compile the PO files (text) to MO files (binary) to get Dactyl to use them.
+1. Create a new subdirectory in the @i18n directory of the portal. For example, to add support for Spanish, create a new subdirectory "es-ES".
 
-To compile all PO files:
+2. Update the i18n configuration in your `redocly.yaml` file defining the display labels for the different languages you support.
 
-```sh
-$ pybabel compile -f -d ./locale/
-```
+        i18n:
+          defaultLocale: en-US
+          locales:
+            - code: en-US
+              name: English
+            - code: ja
+              name: 日本語
+            - code: es-ES
+              name: Spanish
 
-If you added a new language for the first time, you need to make sure its target definition (in the `dactyl-config.yml` file) has the MO file in the `locale_file` field.
+3. Add the translated content in the respective language directory under the @i18n directory. 
 
-After that, next time you build the site using Dactyl it should pull the updated translations!
+    The relative path from the language directory to the translated file should be the same as the relative path from the root of the portal to the file in the default language. For example, if you originally had a file with path `path/to/my/markdown.md`, the file translated to Spanish must be /`@i18n/es-ES/path/to/my/markdown.md`.
```

Comment:
...translated file [must] be the same as the relative path....

[Assuming it's a requirement and not suggested best practice.]